internal/integrations/v4/integration/definition_test.go

```diff
@@ -63,7 +64,9 @@ func TestRun_NoDiscovery(t *testing.T) {
 	require.NoError(t, err)
 
 	// THEN no tasks are executed
-	assert.Empty(t, outs)
+	assert.Equal(t, "stdout line", testhelp.ChannelRead(outs[0].Receive.Stdout))
+	assert.Equal(t, "error line", testhelp.ChannelRead(outs[0].Receive.Stderr))
+	assert.Equal(t, "${discovery.foo}-", testhelp.ChannelRead(outs[0].Receive.Stdout))
```

Comment:
Why does it have `-` in the value 🤔

Reply:
Is the output of `internal/integrations/v4/fixtures/basic cmd.sh`
```
#!/usr/bin/env sh

echo "stdout line"
echo "error line" 1>&2
echo "${PREFIX}-$1"
```